src/pages/proposals/new-cards.tsx

```diff
@@ -0,0 +1,261 @@
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { Pagination } from '@material-ui/lab';
+import styled from '@emotion/styled';
+import { useAppSelector } from 'src/hooks/use-app-select';
+import { RootState } from 'src/reducers/store';
+import { useHistory } from 'react-router-dom';
+import { useVotingDetails } from 'src/hooks/use-voting-details';
+import TooltipGraphSection from './tooltip-graph-section';
+import moment from 'moment';
+
+type CardsProps = {
+	home?: boolean;
+	proposals: any[];
+};
+
+const selector = (state: RootState) => ({
```

Comment:
We should be fetching proposals using the stores api like discussed in the ticket